env/preview.env

```diff
@@ -5,7 +5,7 @@ NODE_ENV=preview
 PORT=8081
 HOST=https://pwabuilder-cloudapk-pre.azurewebsites.net
 
-JDK8PATH=/usr/lib/jvm/java-8-openjdk-amd64/
```

Comment:
since this is no longer using JDK8, perhaps we could rename this environment variable to `JDK17PATH`, or more generally `JDKPATH`

Reply:
Yeah this should be done, I am trying to keep changes to a bare minimum at the moment though for maintenance / timing reasons.